build-system/app.js

```diff
@@ -1192,6 +1192,30 @@ const generateJson = numberOfItems => {
   return results;
 };
 
+const generateResults = (category, count = 2) => {
+  const r = {};
+  const items = [];
+  for (let i = 0; i < count; i++) {
+    const buster = randInt(10000);
+    const item = {};
+    item.src = `https://placeimg.com/600/400/${category}?${buster}`;
+    items.push(item);
+  }
+
+  r.items = items;
+  return r;
+};
+
+app.get('/infinite-scroll-random/:category', function(request, response) {
+  const {category} = request.params;
+  const result = generateResults(category);
+  result['load-more-src'] =
```

Comment:
Should this line be part of `generateResults`?

Reply:
Done